packages/astro/src/cli/index.ts

```diff
@@ -133,6 +138,16 @@ export async function cli(args: string[]) {
 		logging.level = 'silent';
 	}
 
+	if (cmd === 'add') {
+			try {
+				const packages = flags._.slice(3) as string[];
+				await add(packages, { cwd: projectRoot, flags, logging });
```

Comment:
why not run this after loading config? I'd imagine that could also act as a light validation/lint step of your config before we make changes to it.

Also, should we use `projectRoot` or `process.cwd()` always when we run things like `npm install`? I wonder if I'd actually expect it to use `process.cwd()` as a user?